boards/cc2650stk/Makefile.features

```diff
@@ -0,0 +1,9 @@
+FEATURES_PROVIDED += periph_cpuid
```

Comment:
please use the template as in other `Makefile.features` files here, e.g. from `boards/stm32f4discovery/Makefile.features`